blt/blt.yml

```diff
@@ -146,13 +146,13 @@ modules:
       ]
     uninstall:
       [
-        syslog,
+        cgov_syslog,
         cgov_caching_nocdn
       ]
   01live:
     enable:
       [
-        syslog,
+        cgov_syslog,
```

Comment:
Install is fine here because it would trigger the install of all the dependencies of the module, e.g., syslog.